Xplat/src/main/java/artemis/better_climbing/mixin/LivingEntityMixin.java

```diff
@@ -79,7 +81,6 @@ private void better_climbing_incrementClimbTimer(CallbackInfoReturnable<Vec3> ci
 		} else {
 			climbUpTicks = 0;
 		}
-		cir.setReturnValue(movement);
```

Comment:
What's up with this line disappearing? This doesn't look like it would preserve the same behavior.